src/App/Hovedside/Hovedside.tsx

```diff
@@ -41,6 +43,19 @@ class Hovedside extends Component<{}, State> {
             lenketekst={"Gå til rekruttering"}
             lenke={"https://www.nav.no/Forsiden"}
           />
+          <AltinnBoks> </AltinnBoks>
+          <div className={"containerTlfogKontakt"}>
+            <div className={"boks"}>
+              <Undertittel>{"Arbeidsgivertelefonen"}</Undertittel>
+              <a className={"arbTlf"} href="tel:55-55-33-36">
```

Comment:
Finnes en `<Lenke />` i designbiblioteket. Regner med den også kan brukes for tlfnr.
Da kan man fjerne alt av CSS stylingen relatert font